lab-traceyr/README.me

```diff
@@ -0,0 +1,9 @@
+#Instructions
+
+-Once you download the file, in your root directory of this file, open node server.js in your terminal
```

Comment:
Based on your current file structure, it's actually 'node index.js', not 'node server.js'. Moving forward, you may be better off sticking with the '_server.js' and 'server.js' naming scheme as index.js is generally reserved for pulling all of your program files together.